eras/shelley-ma/test-suite/test/Test/Cardano/Ledger/Mary/Examples/MultiAssets.hs

```diff
@@ -122,7 +123,7 @@ policyFailure p =
 outTooBigFailure ::
   ShelleyTxOut MaryEra -> Either (NonEmpty (PredicateFailure (ShelleyLEDGER MaryEra))) (UTxO MaryEra)
 outTooBigFailure out =
-  Left . pure . UtxowFailure . UtxoFailure $ OutputTooBigUTxO [out]
+  Left . pure . UtxowFailure . UtxoFailure $ OutputTooBigUTxO $ NonEmpty.singleton out
```

Comment:
Here is a cute trick that doesn't require an import :wink: 
```suggestion
  Left . pure . UtxowFailure . UtxoFailure . OutputTooBigUTxO $ pure out
```

Reply:
Fair enough, I do like my `singleton`s 😄